### PR TITLE
2nd Milestone Project Calculator, React

### PR DESCRIPTION
**Pull request by AnthonyTC89**
Hello! This milestone has the basic components in the correct folder (src/components): App, Display, ButtonPanel, and Button.

Button components are inside the ButtonPanel.
Display and ButtonPanel components are inside App.
The App component is rendered in the document.
Some styles are added in Display.
The Button component accepts a name in props.
The Display component accepts a result in props with a default value '0'.
The Live Version is in [React Calculator](https://feature-basic-components.d1mt0f332rdrd4.amplifyapp.com/)

### CODE REVIEW

File: .eslintrc.json
Comment (bolah2009) [inline .eslintrc.json]: Let's use the config file here instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json

If you are having issues with the linter free feel to discuss it on this PR. Also you can disable rules in the source code when it is reasonable using inline comments. See https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

File: .stickler.yml
Comment (bolah2009) [inline .stickler.yml]: Please do not ignore the config file. Use this instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml


Is there a reason why you ignoring this config file?

File: src/components/Button.js
Comment (bolah2009) [inline src/components/Button.js]: [Optional] (Also may solve the linter isse)

Let's use a function component instead since this componen tis stateless. Something like this like equally work the same way.

```jsx
const Button = ({ name }) => (
      <button className="Button" type="button">
        {name}
      </button>
)
```

File: src/components/ButtonPanel.js
Comment (bolah2009) [inline src/components/ButtonPanel.js]: [Optional]

As illustrated above, we can refactor this component to be a funtional component instead.

Also, we could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

File: src/components/Display.js
Comment (bolah2009) [inline src/components/Display.js]: Please see previous comments

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi @AnthonyTC89 

Please see comments below. Feel free to leave comments if you don't understand any of the suggestions.

#### :recycle:  Make these changes and submit another code review request. 👍🏾

#### Feel free to reach me on slack if you having issues implementing these features. :relaxed: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: .eslintrc.json
Comment (jofer86) [inline .eslintrc.json]: **[ REQUIRED ]** File is still different as the requirement please do copy the content of the file [HERE](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) and paste it on your file overwriting the content.

**Review by jofer86 — CHANGES_REQUESTED** [review]
### I can see you tackled the changes requested by my fellow Bola. Please just make sure you tackle the change required below, and you should be good to go.

----------------------
Cheers,
Jorge Fernando Rincón Figueroa
@jofer86 on **Slack**

**Review by bolah2009 — COMMENTED** [review]
Good job implementing the suggested changes. 

It's a 👍🏾 from me.

## Please don't merge PR before it is been approved.

**Pull request merged**
